nodes/node_balancer/cmd/clients.go

```diff
@@ -0,0 +1,97 @@
+package cmd
+
+import (
+	"log"
+	"reflect"
+	"time"
+
+	configs "github.com/bugout-dev/moonstream/nodes/node_balancer/configs"
+)
+
+var clientPool ClientPool
```

Comment:
I would prefer a pattern where you have:

```
func CreateClientPool() (*ClientPool, error) {
    ...
}
```

Then call `CreateClientPool` when you start the server and pass client pool reference wherever it is needed.

Reply:
But I need to store this variable globally, still I will need to declare it same way..

Reply:
Can do this way

```
func CreateClientPool() {
	ethereumClientPool.Client = make(map[string]*Client)
	polygonClientPool.Client = make(map[string]*Client)
}
```